src/components/molecules/UserProfilePicture/UserProfilePicture.tsx

```diff
@@ -16,154 +14,108 @@ import {
 
 import { User } from "types/User";
 
-// @debt remove styled-components in favour of using our standard scss patterns
-import * as S from "./UserProfilePicture.styles";
+import { UserReactions } from "./UserReactions";
+
 import "./UserProfilePicture.scss";
 
+const randomAvatarUrl = (id: string) =>
+  "/avatars/" +
+  RANDOM_AVATARS[Math.floor(id?.charCodeAt(0) % RANDOM_AVATARS.length)];
+
+export interface AvatarUrlProps {
+  user: WithId<User>;
+  miniAvatars?: boolean;
+}
+
+const avatarUrl = ({ user, miniAvatars }: AvatarUrlProps): string => {
```

Comment:
```suggestion
const getAvatarUrl = ({ user, miniAvatars }: AvatarUrlProps): string => {
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1323/commits/5efb685f8665173e76d562c8896cae5b5d31cb12